File: DatingApp.Api/Dto/Message.cs
using System;

namespace DatingApp.Api.Dto
{
    public class Message
    {
        public int SenderId { get; set; }
        public int RecipientId { get; set; }
        public DateTime MessageSent { get; set; }
        public string Content { get; set; }

        public Message()
        {
            MessageSent = DateTime.UtcNow;
        }
    }
}

File: DatingApp-Spa/src/app/models/message.ts
export interface Message{
   id: number;
   senderId: number;
   senderPhotoUrl: string;
   senderKnownAs: string;
   recipentId: string;
   recipentKnownAs: string;
   recipentPhotoUrl: string;
   content: string;
   isRead: boolean;
   dateRead: Date;
   messageSent: Date;
}
